packages/functional-tests/pages/resetPassword.ts

```diff
@@ -28,6 +27,14 @@ export class ResetPasswordPage extends BaseLayout {
     return resetPass.isVisible();
   }
 
+  async beginResetPasswordHeadingReact() {
+    const beginResetPasswordHeading = this.page.getByRole('heading', {
```

Comment:
Not really sure how I feel about this. It seems like it is a completely new page and might be btter in its own class?

Reply:
New class for the React version of this page*